l1-contracts/deploy-scripts/LensScript.s.sol

```diff
@@ -0,0 +1,371 @@
+        console.log("tempProxyAdmin:", config.tempProxyAdmin);
+        console.log("tempValidatorTimelock:", config.tempValidatorTimelock);
+
+        console.log("tempL1SharedBridge:", config.tempL1SharedBridge);
+        console.log("tempErc20Bridge:", config.tempErc20Bridge);
+
+        console.log("newStateTransitionManager:", config.newStateTransitionManager);
+        console.log("newBridgehub:", config.newBridgehub);
+        console.log("newProxyAdmin:", config.newProxyAdmin);
+        console.log("newValidatorTimelock:", config.newValidatorTimelock);
+        console.log("newVerifier:", config.newVerifier);
+        console.log();
+    }
+
+    function _generateTempEcoOwnershipCalls() internal {
+        IGovernance.Call[] memory tempEcoOwnershipCalls = new IGovernance.Call[](11);
+        // proxy admin - bh
+        tempEcoOwnershipCalls[0] = IGovernance.Call({
+            target: config.tempProxyAdmin,
+            value: 0,
+            data: abi.encodeCall(
+                ProxyAdmin.changeProxyAdmin,
+                (ITransparentUpgradeableProxy(config.tempBridgehub), config.newProxyAdmin)
+            )
+        });
+
+        // proxy admin - stm
+        tempEcoOwnershipCalls[1] = IGovernance.Call({
+            target: config.tempProxyAdmin,
+            value: 0,
+            data: abi.encodeCall(
+                ProxyAdmin.changeProxyAdmin,
+                (ITransparentUpgradeableProxy(config.tempStateTransitionManager), config.newProxyAdmin)
+            )
+        });
+
+        // proxy admin - l1 shared bridge
+        tempEcoOwnershipCalls[2] = IGovernance.Call({
+            target: config.tempProxyAdmin,
+            value: 0,
+            data: abi.encodeCall(
+                ProxyAdmin.changeProxyAdmin,
+                (ITransparentUpgradeableProxy(config.tempL1SharedBridge), config.newProxyAdmin)
+            )
+        });
+
+        // proxy admin - erc20 bridge
+        tempEcoOwnershipCalls[3] = IGovernance.Call({
+            target: config.tempProxyAdmin,
+            value: 0,
+            data: abi.encodeCall(
+                ProxyAdmin.changeProxyAdmin,
+                (ITransparentUpgradeableProxy(config.tempErc20Bridge), config.newProxyAdmin)
+            )
+        });
+
+        // bridgehub - transfer ownership
+        tempEcoOwnershipCalls[4] = IGovernance.Call({
+            target: config.tempBridgehub,
+            value: 0,
+            data: abi.encodeCall(Ownable2Step.transferOwnership, (transitionaryOwner))
+        });
+
+        // state transition manager - transfer ownership
+        tempEcoOwnershipCalls[5] = IGovernance.Call({
+            target: config.tempStateTransitionManager,
+            value: 0,
+            data: abi.encodeCall(Ownable2Step.transferOwnership, (transitionaryOwner))
+        });
+
+        // validator timelock - transfer ownership
+        vtOwnershipCall = abi.encodeWithSelector(Ownable2Step.transferOwnership.selector, transitionaryOwner);
```

Comment:
I understand that the setup is a bit broken, but in this case, it is better to also print this call separately